prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -166,13 +150,11 @@ presubmits: # runs on PRs
       preset-build-release: "true"
       <<: *gke_job_labels_template
     extra_refs:
-    - org: mszostok
```

Comment:
test-infra-ref can be used

Reply:
cannot be used because `base_ref` is defferent, `master` -> `release-0.7`

Reply:
my mistake, fixed